### PR TITLE
Sceau 2.13

### PR DESCRIPTION
**Pull request by yanncyrille**
test/prod mode distinction
test orders can be resend

### CODE REVIEW

File: fianetsceau.php [hunk 1]
```diff
@@ -471,9 +486,10 @@ public function sendXML($id_order)
 				else
 					$image_url = null;
 			}
-			$image_url = null;
-			$product_name = str_replace("'", '', $product['product_name']);
-			$product_name = str_replace('&', '', $product_name);
+			$image_url = null;	
+			
+			$product_name = str_replace(array('&', "'", '�'), array('', '', ''), $product['product_name']);
```
Comment (Quetzacoalt91) [inline fianetsceau.php:491]: Please have a look at the [module validator](https://validator.prestashop.com). Your file at the wrong encoding, and should be in utf-8

**Comment by Quetzacoalt91** [comment]
Hi @yanncyrille,

Because you modify one of your table structure, you have to alter it for the merchants who has already installed the module with an upgrade file.

Please see this link to have more information: http://doc.prestashop.com/display/PS15/Auto-updating+modules

**Comment by yanncyrille** [comment]
Hi @Quetzacoalt91,
unless I am mistaken, auto-updating modules method is only available for PS1.5 and many of our merchants are still using lower version 1.5

**Comment by Quetzacoalt91** [comment]
Hi @yanncyrille,

Yes, you are right, but you need to manage this upgrade or your merchant will get an error saying a column is missing !
For the merchant using PrestaShop, you need to manage manually this upgrade. Here is an example which can be found in another module: https://github.com/PrestaShop/ganalytics/blob/aa9430712ed92fb26f220f2d2b1329b146bda041/ganalytics.php#L772-L785

**Comment by yanncyrille** [comment]
Hi @Quetzacoalt91,

Merchants will not have the error because the method "checkSceauUpdate" checks the version of the module and reinstall it if the version is not good.
When reinstalling the module, a full backup of the tables is made and they are replaced by new tables with new columns.

Moreover, it will be impossible to test the method "upgrade_module_2_13" until the next update of the module. How to be sure that this method will be triggered when the merchant clicks on "Update"?

**Comment by gaillafr** [comment]
Right after installing the module I can see that temporary table has not been created: https://www.dropbox.com/s/q6q1ovji35tkywn/Screenshot%202015-06-08%2012.10.19.png?dl=0

**Comment by yanncyrille** [comment]
This message is displayed if the module does not need to save old orders. During an initial installation, the tables do not exist, so it is not necessary to back up existing orders so no need to create temporary tables. This message does not prevent the installation of the module. It is purely informative.

**Comment by gaillafr** [comment]
Thank you @yanncyrille, your PR has been manually merged on the 'dev' branch: https://github.com/PrestaShop/fianetsceau/commit/aaed1b93099cb27b4f26ea90a1e62f0e901d615e

**Pull request closed without merging**